source/features/mark-unread.tsx

```diff
@@ -49,7 +49,7 @@ async function getNotifications(): Promise<Notification[]> {
 	const {unreadNotifications} = await browser.storage.local.get({
 		unreadNotifications: []
 	});
-	return unreadNotifications;
+	return unreadNotifications.filter(({url}: AnyObject) => location.hostname.endsWith(new URL(url).hostname));
```

Comment:
That's a good solution

```suggestion
	return unreadNotifications.filter(({url}: Notification) => location.hostname.endsWith(new URL(url).hostname));
```